invertedai/large/common.py

```diff
@@ -0,0 +1,153 @@
+from typing import List, Optional, Tuple
+from pydantic import BaseModel
+
+from invertedai.common import AgentAttributes, AgentState, RecurrentState, Point
+
+class Region(BaseModel):
```

Comment:
If we only allow squares for the time being, it's better to remove all the currently unused fields. They serve no purpose currently and only commit you to some design choices that may turn out to be suboptimal in the future. You can always add them later. Similarly, making things optional in the future is a non-breaking change, so if the value for a given field is currently required, don't make it optional. I would remove everything except `centre` and the agent specifications.

Reply:
I've kept size (formerly fov) for now since that is necessary for region_drive but otherwise everything else has been removed.